### ISSUE TITLE
'Reset Flight' resets persistent timers

### ISSUE THREAD

**Issue by HaroldAnders** [FrSkyRC/ETHOS-Feedback-Community#3325]
The persistent timer works with Ethos except when using the [long press] 'Reset flight' between flights.  If you turn off the TX the flight time is preserved as expected. I installed Ver 1.4.17 to make sure it hasn't been updated. Not sure if this is a bug or a feature...  

I used Timer3 with persistent enabled in OTX to track my total flight times on planes.   This worked fine as long as I didn't inadvertently press  'Reset Timers'.

**Comment by bsongis-frsky** [comment]
"Reset Flight" should reset the persistent timers

**Comment by flyingeek** [comment]
I agree with HaroldAnders Reset Flight should not reset persistent timers. A Reset Model should, a Reset Flight not.

**Comment by HaroldAnders** [comment]
The boss has spoken.  What would have been a bug in OpenTX is now a *feature* in Ethos.  It's too bad as I track total hours of flight time on my aircraft but I now have several planes now with over 40 hours of 'throttle on' flight time on my X20.  You just have to set up a reset button for timers 1 and 2.  It is not a big deal, actually.   Just don't forget and do a 'reset flight' because this now actually means 'reset the world'.

**Comment by bsongis-frsky** [comment]
The issue has been closed automatically. The debate is not closed.
I think the request is valid, but I will have difficulties changing the current behaviour of Reset Flight.
Perhaps `persistent` could have more values than just true / false, like:
Reset timer on:
- [X] Model load
- [X] Reset flight
- [X] ...

**Comment by flyingeek** [comment]
Why not consider persistent as an "immutable" value if timer reset configuration is '---' . By immutable I mean that request must be done explicitly for this timer, either by the contextual menu in the Timers page, or by using the Reset Timer contextual menu of the Timer Value widget.

If a user wants Ethos to perform a reset of this timer he could then specify it in the reset condition (ie Flight Reset)

If a user wants a reset on Model load, then this is the normal behaviour, just don't make it persistent.

Ethos is very powerful with timers, even more with the Timer Logs widget, I track complicated things especially with gliders, reset can be made very specifically and I never found a use case of persistent timer which should be reset at any time.

I also think that the Reset Timers in the contextual widget should not reset permanent timers, thanks to this simple logic. It would be quite simple to explain it in the documentation:

Concerning the reset menu:

Reset flight will reset .... , the non persistent timers, ....
Reset timers will reset the non persistent timers.

In the Timers Page:

Persistent
simply add: Ethos will never reset a permanent timer unless a reset condition is defined. You can reset a persistent timer manually using the contextual menu.

**Comment by flyingeek** [comment]
Ok to keep this issue alive, a link to edgetx handling of this problem: https://manual.edgetx.org/color-radios/model-settings/model-setup/timer-1-2-3

Excerpt:

**Persistent**:

**Off** - The timer value is reset when switching models or when the radio is turned off / on.

**Flight** - The timer value is NOT reset when switching models or when radio is turned off / on. The timer value is only reset when the Reset flight option is selected in the [Reset telemetry ](https://manual.edgetx.org/color-radios/reset-telemetry)menu.

**Manual Reset** - The timer value is reset only when it is individually selected to be reset (example: Reset timer1) in the [Reset telemetry ](https://manual.edgetx.org/color-radios/reset-telemetry)menu.

_The Flight persistent setting can be set for multiple timers and then these timers can be reset simultaneously with the Reset flight option._

**Comment by flyingeek** [comment]
still needed
